tests/data/fitsheader-tuple.yaml

```diff
@@ -0,0 +1,146 @@
+!<lsst.daf.base.PropertyList>
+- !!python/tuple [SIMPLE, Bool, true, '']
```

Comment:
Should I be concerned that each entry starts with `python/tuple` but is a list, not a tuple?